Misc/NEWS

```diff
@@ -291,6 +291,9 @@ Extension Modules
 Library
 -------
 
+- bpo-21071: struct.Struct.format is now a Unicode instead, instead of a bytes
```

Comment:
Unicode instead → Unicode string
bytes string → byte string [or bytes object if you prefer]